sklearn/covariance/_graph_lasso.py

```diff
@@ -482,9 +485,7 @@ class GraphicalLasso(BaseGraphicalLasso):
     GraphicalLassoCV : Sparse inverse covariance with
         cross-validated choice of the l1 penalty.
 
-    For a usage example see
-    :ref:`sphx_glr_auto_examples_applications_plot_stock_market.py`
-
+    
```

Comment:
This extra line still has to be removed, it is causing the linting to fail due to a blank space.

Reply:
ok, I already delete . Sorry to bother~